src/EnergyPlus/DualDuct.hh

```diff
@@ -77,125 +73,84 @@ namespace DualDuct {
         Num
     };
 
-    enum class DualDuctOAMode
-    {
-        Invalid = -1,
-        ConstantOAMode = 11,
-        ScheduleOAMode = 12,
-        DynamicOAMode = 13,
-        Num
-    };
-
     enum class PerPersonMode
     {
         Invalid = -1,
-        ModeNotSet = 20,
-        DCVByCurrentLevel = 21,
-        ByDesignLevel = 22,
+        ModeNotSet,
+        DCVByCurrentLevel,
+        ByDesignLevel,
         Num
     };
 
-    // DERIVED TYPE DEFINITIONS
-
-    // MODULE VARIABLE DECLARATIONS:
-
     struct DualDuctAirTerminalFlowConditions
     {
-        // Members
-        Real64 AirMassFlowRate;         // MassFlow through the Damper being Simulated [kg/Sec]
-        Real64 AirMassFlowRateMaxAvail; // MassFlow through the Damper being Simulated [kg/Sec]
-        Real64 AirMassFlowRateMinAvail; // MassFlow through the Damper being Simulated [kg/Sec]
-        Real64 AirMassFlowRateMax;      // Max Mass Flow Rate or the Design Mass Flow Rate
-        Real64 AirTemp;
-        Real64 AirHumRat;
-        Real64 AirEnthalpy;
-        Real64 AirMassFlowRateHist1; // flow history back 1 iteration kg/s
-        Real64 AirMassFlowRateHist2; // flow history back 2 iteration kg/s
-        Real64 AirMassFlowRateHist3; // flow history back 3 iteration kg/s
-        Real64 AirMassFlowDiffMag;   // flow difference scale, kg/s
-
-        // Default Constructor
-        DualDuctAirTerminalFlowConditions()
-            : AirMassFlowRate(0.0), AirMassFlowRateMaxAvail(0.0), AirMassFlowRateMinAvail(0.0), AirMassFlowRateMax(0.0), AirTemp(0.0), AirHumRat(0.0),
-              AirEnthalpy(0.0), AirMassFlowRateHist1(0.0), AirMassFlowRateHist2(0.0), AirMassFlowRateHist3(0.0), AirMassFlowDiffMag(0.0)
-        {
-        }
+        Real64 AirMassFlowRate = 0.0;         // MassFlow through the Damper being Simulated [kg/Sec]
+        Real64 AirMassFlowRateMaxAvail = 0.0; // MassFlow through the Damper being Simulated [kg/Sec]
+        Real64 AirMassFlowRateMinAvail = 0.0; // MassFlow through the Damper being Simulated [kg/Sec]
+        Real64 AirMassFlowRateMax = 0.0;      // Max Mass Flow Rate or the Design Mass Flow Rate
+        Real64 AirTemp = 0.0;
+        Real64 AirHumRat = 0.0;
+        Real64 AirEnthalpy = 0.0;
+        Real64 AirMassFlowRateHist1 = 0.0; // flow history back 1 iteration kg/s
+        Real64 AirMassFlowRateHist2 = 0.0; // flow history back 2 iteration kg/s
+        Real64 AirMassFlowRateHist3 = 0.0; // flow history back 3 iteration kg/s
+        Real64 AirMassFlowDiffMag = 0.0;   // flow difference scale, kg/s
```

Comment:
Two things about this list of fields. First, every field starts with `Air` suggesting that word should appear in the name of the `struct` instead and it does! Second, I have noticed that the quad `Temp`, `HumRat`, `MassFlowRate`, and `Enthalpy` appears in many places. This suggests it itself should be a `struct` so that it is name and used consistently and that these four values can be passed to functions using a single reference. This is what we did with `PlantLocation`.